src/python/WMComponent/TaskArchiver/DataCache.py

```diff
@@ -0,0 +1,12 @@
+import time
+
+class DataCache(object): 
+    _finishedWFs = [] # Global cache holding workflow lists 
+    
+    @staticmethod
+    def getFinishedWorkflows():
+        return DataCache._finishedWFs;
+
+    @staticmethod
+    def setFinishedWorkflows(workflowLists):
+        DataCache._finishedWFs = workflowLists;
```

Comment:
this will overwrite the cache. How about we use a set() for the class attr and then we update() here?
Then, in the get method it could cast to a list if needed.